nifi-nar-bundles/nifi-hive-bundle/nifi-hive-processors/pom.xml

```diff
@@ -85,6 +85,12 @@
                 </exclusion>
             </exclusions>
         </dependency>
+        <!-- Override groovy-all:2.1.6 from Hive -->
+        <dependency>
+            <groupId>org.codehaus.groovy</groupId>
+            <artifactId>groovy-all</artifactId>
+            <version>2.4.21</version>
```

Comment:
Why are we leaving it at 2.4.X here? Breaking change in 2.5?

Reply:
Thanks for the feedback @MikeThomsen! The main reason for staying with the 2.4 series in the Hive processors was to minimize the impact for those old components. Hive3 depends on Groovy 2.4, the others depend on earlier versions. It is a transitive dependency through `hive-exec`, and it may be acceptable to move up to 2.5, but staying with 2.4 seemed acceptable in this case since it also addresses the underlying issues.